api/README.md

```diff
@@ -0,0 +1,135 @@
+### Summary
+
+This example shows how, having a screen, attach to it a newly created asset with js injection.
+Here are some examples and tips on writing a proper js injection code: * [Js Injection](../javascript-injectors/README.md)
+
+For this you would need an API token(Can be found on web dashboard /manage/account/team)
```

Comment:
This is an invalid sentence. Please run these sentences through [Grammarly](https://www.grammarly.com/) or similar.